conf/cndp_upf_1worker.jsonc

```diff
@@ -0,0 +1,168 @@
+{
+    // (R) - Required entry
+    // (O) - Optional entry
+    // All descriptions are optional and short form is 'desc'
+    // The order of the entries in this file are handled when it is parsed and the
+    // entries can be in any order.
+
+    // (R) Application information
+    //    name        - (O) The name of the application
+    //    description - (O) The description of the application
+    "application": {
+        "name": "cndp-upf",
+        "description": "CNDP OMEC-UPF configuration"
+    },
+
+    // (O) Default values
+    //    bufcnt - (O) UMEM default buffer count in 1K increments
+    //    bufsz  - (O) UMEM buffer size in 1K increments
+    //    rxdesc - (O) Number of RX ring descriptors in 1K increments
+    //    txdesc - (O) Number of TX ring descriptors in 1K increments
+    //    cache  - (O) MBUF Pool cache size in number of entries
+    //    mtype  - (O) Memory type for mmap allocations
+    "defaults": {
+        "bufcnt": 16,
```

Comment:
Any reason why the default `bufcnt` here is 16 and it is 32 in the other two files (4worker and 8worker)?

Reply:
bufcnt in umem section override this value. I can keep it to 32 for consistency